opl_storybook.features.field_instance.inc

```diff
@@ -100,8 +100,8 @@ function opl_storybook_field_default_field_instances() {
       'description_field' => 0,
       'entity_translation_sync' => 0,
       'file_directory' => 'media_uploads',
-      'file_extensions' => 'mpeg ogg wav',
-      'max_filesize' => '',
+      'file_extensions' => 'mpeg ogg wav mp3',
+      'max_filesize' => '6 MB',
```

Comment:
Make sure that the php.ini setting on the webserver will allow a 6mb file to be uploaded.